src/main/java/liquibase/ext/cassandra/snapshot/ForeignKeySnapshotGeneratorCassandra.java

```diff
@@ -0,0 +1,29 @@
+package liquibase.ext.cassandra.snapshot;
+
+import liquibase.database.Database;
+import liquibase.ext.cassandra.database.CassandraDatabase;
+import liquibase.snapshot.DatabaseSnapshot;
+import liquibase.snapshot.jvm.ForeignKeySnapshotGenerator;
+import liquibase.structure.DatabaseObject;
+
+public class ForeignKeySnapshotGeneratorCassandra extends ForeignKeySnapshotGenerator {
```

Comment:
As Cassandra doesn't support foreignKeys, the problem is not in SnapshotGenerator.
Adding or dropping foreign keys will fail anyway when default implementation from liquibase-core will generate SQL that fails to execute, without snapshotting logic ever involved.
I think the best way to handle is to override supports method in CassandraDatabase class
```
    @Override
    public boolean supports(Class<? extends DatabaseObject> object) {
        if (ForeignKey.class.isAssignableFrom(object)) {
            return false;
        }
        return super.supports(object);
    }

 ```

Reply:
great. let me try it.